core/src/main/scala/com/yahoo/maha/core/FilterOperation.scala

```diff
@@ -1223,7 +1225,14 @@ object Filter extends Logging {
       case OuterFilter(filters) =>
         makeObj(
            ("operator" -> toJSON(filter.operator.toString))
-            :: ("outerFilters" -> toJSON(filters))
+            :: ("outerFilters" -> {
+             val values: mutable.ListBuffer[JValue] = mutable.ListBuffer.empty[JValue]
```

Comment:
do you still need to do this if they are still List[_] ?  wouldn't toJSON(filters)) work ?